src/common/styles/tests.styles.ts

```diff
@@ -72,12 +72,37 @@ export const backContainerSentence = css`
 export const homeBackContainer = css`
   margin-top: ${spacing(19)};
   margin-bottom: ${spacing(6)};
-  padding: ${spacing(10)} ${spacing(5)} ${spacing(1)};
+  padding: ${spacing(0)} ${spacing(0)} ${spacing(1)};
   min-height: 20rem;
   position: relative;
   padding-bottom: 1.5rem;
   background-color: ${color.lightWhite};
-  border-radius: 2rem;
+`;
+
+export const headerHome = css`
+  padding: ${spacing(1)} ${spacing(1.5)};
+  display: flex;
+  // flex-direction: row;
```

Comment:
remove commented code (lechuguitas)

Reply:
@brauliodiez I just deleted the comments